cpp/include/cucim/core/framework.h

```diff
@@ -63,7 +63,7 @@ struct PluginLoadingDesc
 
 struct Framework
 {
-    void load_plugins(const PluginLoadingDesc& desc = PluginLoadingDesc::get_default());
+    // void load_plugins(const PluginLoadingDesc& desc = PluginLoadingDesc::get_default());
```

Comment:
Should we drop this? Or if we want to add this back later, should we add an explanatory comment/issue to track?

Reply:
Updated a comment and created an issue for handling that.
- https://github.com/rapidsai/cucim/issues/134